frontend/src/components/ProcessContent.tsx

```diff
@@ -0,0 +1,59 @@
+import { useTranslation } from "next-i18next";
+import { Grid, GridContainer } from "@trussworks/react-uswds";
+
+const ProcessContent = () => {
+  const { t } = useTranslation("common", { keyPrefix: "Process" });
+
+  return (
+    <GridContainer className="padding-bottom-5 tablet:padding-top-0 desktop-lg:padding-top-0 border-bottom-2px border-base-lightest">
+      <h1 className="margin-0 tablet-lg:font-sans-xl desktop-lg:font-sans-2xl">
+        {t("title")}
+      </h1>
+      <Grid row gap>
+        <Grid
+          tabletLg={{ col: 12 }}
+          desktop={{ col: 12 }}
+          desktopLg={{ col: 12 }}
+        >
+          <p className="tablet-lg:font-sans-xl line-height-sans-3 usa-intro margin-top-2">
+            {t("content")}
+          </p>
+        </Grid>
+      </Grid>
+      <Grid row gap className="flex-align-start">
```

Comment:
Sammy and I talked about how this bit might be a reusable component, as there's a similar pattern of boxes on the research page.

Reply:
But this is probably fine for now. We can swap to a component later.